### ISSUE TITLE
regs_access() for RISC-V yield error

### ISSUE THREAD

**Issue by KelvinChung2000** [aquynh/capstone#2406]
### Work environment

| Questions                                | Answers
|------------------------------------------|--------------------
| OS/arch/bits                             | Pop OS
| Architecture                             | x86
| Source of Capstone                       | pip
| Version/git commit                       | v5.0.1

### Expected behavior
`regs_access()` should return the read and write regs 

### Actual behavior
Rasing error on unsupported architecture

### Steps to reproduce the behavior

```Python
from capstone import Cs
from capstone import CS_ARCH_RISCV, CS_MODE_RISCV64

CODE = b"\x83\x3e\x05\x08"
md = Cs(CS_ARCH_RISCV, CS_MODE_RISCV64)
for insn in md.disasm(CODE, 0):
    print(insn.bytes)
    print(f"{insn.mnemonic} {insn.op_str}")
    print(insn.regs_access())
```

### Additional Logs, screenshots, source code,  configuration dump, ...
![image](https://github.com/user-attachments/assets/21865235-7637-4fb3-9be1-8f79d9220f30)

**Comment by Rot127** [comment]
@moste00 The register access is here still missing, not just for Python:
```
cstool -d riscv64 "0x83, 0x3e, 0x05, 0x08"
 0  83 3e 05 08  ld	t4, 0x80(a0)
	ID: 210 (ld)
	op_count: 2
		operands[0].type: REG = t4
		operands[0].access: WRITE
		operands[1].type: MEM
			operands[1].mem.base: REG = a0
			operands[1].mem.disp: 0x80
		operands[1].access: READ

	Groups: isrv64 
```

**Comment by moste00** [comment]
@Rot127 Hmm, does this output count as missing though ? there are two registers in the instructions and they show in the details with their correct read/write property, do you mean to say that `a0` should appear as its own operand rather than as the base for the mem operand ?

**Comment by Rot127** [comment]
Yes, it is missing because the `cs_struct.reg_access` callback is not implemented.
